src/components/TokenClaimDialog.tsx

```diff
@@ -453,12 +456,24 @@ function useRewardClaims() {
         rewards: GaugeReward[]
       } | null,
     ) => {
-      if (!chainId || !account || !gaugeMinterContract || !library || !gauge)
+      if (!chainId || !account || !gaugeMinterContract || !library || !gauge) {
+        enqueueToast("error", "Unable to claim reward")
+        return
+      }
+
+      if (gauge.rewards.length === 0) {
+        enqueueToast("error", "No rewards to claim")
         return
+      }
       try {
         updateClaimStatus(gauge.gaugeName, STATUSES.PENDING)
         const claimPromises = []
-        if (gauge.rewards.length > 0 && gauge.address) {
+        const minterRewards = gauge.rewards.filter(({ isSDL }) => isSDL)
```

Comment:
Can we update the var names to `sdlRewards` and `externalRewards` to make things clearer? Anything helps since the entire concept is pretty confusing anyway.